db/migrate/20250819132430_add_local_authority_postcodes.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddLocalAuthorityPostcodes < ActiveRecord::Migration[8.0]
+  def change
+    create_table :local_authority_postcodes,
+                 id: false,
+                 primary_key: %i[gss_code postcode] do |t|
+      t.string :gss_code, null: false
+      t.string :postcode, null: false
```

Comment:
I'm wondering if `value` is a better name here to avoid code like `postcode.postcode`. It's [similar to what we do for `SessionDates`](https://github.com/nhsuk/manage-vaccinations-in-schools/blob/906e02eb24a9a3b04af6b25ef2d5e4d6cd33438b/app/models/session_date.rb#L8).